### PR TITLE
fix: prompt copywriting optimization of enterprise blocking

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
prompt copywriting optimization of enterprise blocking

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.3

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/locales/zh.json [hunk 1]
```diff
@@ -2554,6 +2552,7 @@
     "namespace": "命名空间",
     "need to configure project cluster resources": "需要配置项目集群资源",
     "network billing method": "网络付费方式",
+    "network closure within the scope of the organization, Precautions for network closure. If urgent release and upgrading operations are required, the organization administrator must review and authorize them.": "封网是组织范围内，禁止指定环境下所有应用进行部署、重启、升级等相关操作，如需紧急发布、升级操作都需要组织管理员审核授权。",
```
Comment (daskyrk) [inline shell/app/locales/zh.json:2555]: Use short key words as key

Comment (wuhuiluo) [reply shell/app/locales/zh.json]: ok

File: shell/app/locales/en.json [hunk 2]
```diff
@@ -2554,6 +2552,7 @@
     "namespace": "namespace",
     "need to configure project cluster resources": "need to configure project cluster resources",
     "network billing method": "network billing method",
+    "definition of network closure": "definition of network closure",
```
Comment (daskyrk) [inline shell/app/locales/en.json:2555]: Shorten the key but not the value.

Comment (wuhuiluo) [reply shell/app/locales/en.json]: ok,Modified

**Pull request merged**